pkg/util/cert/io.go

```diff
@@ -25,14 +25,25 @@ import (
 	"path/filepath"
 )
 
-// CanReadCertOrKey returns true if the certificate or key files already exists,
+// CanReadCertAndKey returns true if the certificate and key files already exists,
 // otherwise returns false.
-func CanReadCertOrKey(certPath, keyPath string) bool {
-	if canReadFile(certPath) || canReadFile(keyPath) {
-		return true
+func CanReadCertAndKey(certPath, keyPath string) (bool, error) {
+	certExist := canReadFile(certPath)
+	keyExist := canReadFile(keyPath)
+
+	if certExist == false && keyExist == false {
+		return false, nil
+	}
+
+	if certExist == false {
```

Comment:
Maybe we can even inline this code. There is no added value of the whole function IMO.